quesma/quesma/search.go

```diff
@@ -235,10 +235,12 @@ func (q *QueryRunner) HandleMultiSearch(ctx context.Context, defaultIndexName st
 	}
 	logger.DebugWithCtx(ctx).Msgf("handling multisearch: queries=%d, indices=[%s], defaultIndex=[%s]", len(queries), queriedIndices, defaultIndexName)
 	for _, query := range queries {
-
-		// TODO ask table resolver here and go to the right connector or connectors
-
-		responseBody, err := q.HandleSearch(ctx, query.indexName, query.query)
+		var responseBody []byte
```

Comment:
Maybe we should add comments why this code is here and not somewhere else with note that its tech dept.